src/components/Header/Header.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useCallback, useState } from 'react';
+import { UnsupportedChainIdError, useStarknetReact } from '@web3-starknet-react/core';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { SvgIcon } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import Box from '@mui/material/Box';
+import { Link, NavLink, useLocation } from 'react-router-dom';
+
+import { HeaderContainer, HeaderLogo, HeaderWallet, AccountElement, Web3StatusConnected, Web3StatusError, Web3StatusConnect, ActiveLink } from './Header.styles';
+import { useActiveStarknetReact } from '../../hooks';
+import { eventsLookup, NETWORK_LABELS } from '../../common/contansts';
+import { argentX, braavosWallet } from '../../common/connectors';
+import { getShortenAddress } from '../../common/addressHelper';
+import { EventEmitter } from '../../common/eventEmitter';
+import logo from '../../resources/images/logo.png';
+import argentXIcon from '../../resources/icons/argentx.svg';
+import braavosIcon from '../../resources/icons/braavos.svg';
+import GradientButton from '../GradientButton/GradientButton';
+import WalletModal from '../../features/wallet/WalletModal/WalletModal';
+import LanguageSwitcher from '../LanguageSwitcher/LanguageSwitcher';
+
+const noop = () => { };
+
+const Header = () => {
+  const { chainId } = useStarknetReact();
+  const location = useLocation();
+
+  return (
+    <HeaderContainer py={1}>
+      <Grid container columnSpacing={{ md: 2 }} alignItems="center">
+        <Grid item xs={6} sx={{ display: 'flex', alignItems: 'center' }}>
+          <HeaderLogo to="/home">
+            <div>
+              <svg width="88" height="20" viewBox="0 0 88 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
please save this svg as a file and include it like here, for example - https://github.com/jediswaplabs/personal-profile/blob/master/src/components/Header/Header.jsx#L17